card/config/initializers/02_patches/active_record.rb

```diff
@@ -0,0 +1,65 @@
+# pluck_in_batches:  yields an array of *columns that is at least size
+#                    batch_size to a block.
+#
+#                    Special case: if there is only one column selected than each batch
+#                                  will yield an array of columns like [:column, :column, ...]
+#                                  rather than [[:column], [:column], ...]
+# Arguments
+#   columns      ->  an arbitrary selection of columns found on the table.
+#   batch_size   ->  How many items to pluck at a time
+#   &block       ->  A block that processes an array of returned columns.
+#                    Array is, at most, size batch_size
+#
+# Returns
+#   nothing is returned from the function
+
+module Patches
```

Comment:
I misspelled it "badges" in the branch name but patches is correct. It's a patch for ActiveRecord.